.github/workflows/ansible-test.yml

```diff
@@ -7,15 +7,15 @@ on:
     # * is a special character in YAML so you have to quote this string
     - cron: '0 7 * * *'
 env:
-  python_version: '3.10'
+  python_version: '3.11'
```

Comment:
Can we use 3.12 or 3.13?

Reply:
We have been using the minimum required version of python here for now. It will get upgraded when a new version of Ansible stops supporting the Python version. I would be nice to use the latest python but it will be hard for us to keep up with the releases.